shell/app/config-page/components/grid/grid.tsx

```diff
@@ -22,12 +22,14 @@ export default (props: CP_GRID.Props) => {
     itemSpan = new Array(children.length).fill(Math.ceil(24 / children.length));
   }
   return (
-    <Row gutter={gutter}>
-      {children.map((child, i) => (
-        <Col span={itemSpan[i]} key={i}>
-          {child}
-        </Col>
-      ))}
-    </Row>
+    <div className="overflow-hidden">
+      <Row gutter={gutter}>
```

Comment:
Why not add className to Row?

Reply:
it's useless add to row, suppose to be forbidden overflow out of Row